examples/new-state/src/lib.rs

```diff
@@ -167,7 +167,7 @@ mod tests {
     fn test_init() {
         let ctx = InitContextTest::empty();
 
-        let state_ll = Rc::new(RefCell::new(ContractStateLLTest::new()));
+        let state_ll = Rc::new(RefCell::new(ContractStateLLTest::default()));
```

Comment:
I strongly prefer being explicit instead of implicit, and default is implicit. Is there are particular reason for this change?